app/views/reports/instrument_reports/report_table.html.haml

```diff
@@ -0,0 +1,20 @@
+-# TODO: this is identical to instrument_day_reports/report_table
+
+.updated_values{style: "display: none"}
+  .date_start= human_datetime(@date_start, date_only: true)
+  .date_end= human_datetime(@date_end, date_only: true)
+
+.export_raw{data: { visible: "false" } }
```

Comment:
Is this line the only difference between these and `reports/report_table`? Would it be cleaner to set an instance variable in the controller to decide whether to be visible or not and use the same partial everywhere? It would be nice to be able to move `xhr_html_template` back to private.